eas-hooks/eas-build-on-success.sh

```diff
@@ -17,20 +33,23 @@ if [[ "$EAS_BUILD_PLATFORM" == "android" ]]; then
     counter=$((counter + 1))
   done
 
+  sleep 10
```

Comment:
What problem is this sleep solving? Every time I see sth like this I immediately think this is some kind of hack. I think this requires at least a comment.

Reply:
This is what we are doing in updates E2E to help ensure that the Android emulator is ready. Probably not needed here, agreed.

Reply:
```suggestion
```